doc/node.1

```diff
@@ -1,829 +1,133 @@
-.TH NODE.JS "1" "2010" "" ""
+.TH NODE.JS 1 2016
 
+.\ This is a man page comment.
+.\ Man syntax is somewhat obscure, but the important part is is that .<letter>
+.\ specifies <letter>'s syntax for that line, and \f<letter> specifies it for
+.\ the characters that follow.
+.\ See http://liw.fi/manpages/ for more info.
 
-.SH "NAME"
-node \- Server-side JavaScript
+.SH NAME
+
+node \- Server-side JavaScript Runtime
 
-.SH SYNOPSIS
 
+.SH SYNOPSIS
 
 .B node
-[
-.B \-v
-]
-[
-.B \-\-debug
-|
-.B \-\-debug-brk
-]
-[
-.B \-\-v8-options
-]
+[\fIoptions\fR] [\fIv8 options\fR]
+[\fB\-e \fR"\fIscript\fR" |\fI script.js\fR]
+[\fIarguments\fR]
 .br
-     [
-.B \-e
-.I command
-|
-.I script.js
-]
-[
-.I arguments
-]
+.B node
+[\fBdebug \fR|\fB \-\-\fR\fBdebug \fR|\fB \-\-debug-brk\fR] \fI...
```

Comment:
@indutny How much of this should we list here? I'm not really sure how the debug options work. :/